### PR TITLE
As2-6 PUT positive update a board

### PR DESCRIPTION
**Pull request by Goldbe54**
Created a test to change the board parameters, wrote the putBoardInfo method in the BoardRestTestClient class

### CODE REVIEW

File: src/test/resources/application.conf [hunk 1]
```diff
@@ -1,2 +1,2 @@
-key="7f07eabdc97e9555e6cfc283a3a3fb9e"
-token="ATTA1418663ae4e3999c60e6d04258e61619af886960da297bf6709f56d1e631ff1084784FA0"
```
Comment (SoniaMiach) [inline src/test/resources/application.conf:2]: the same problem 
change token and key to the previous

File: src/test/java/api/put/UpdateABoard.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class UpdateABoard extends BaseTest {
+    private String NAME_VALUE = "Test board";
+    private String DESC_VALUE = "Decription is empty";
+    private String CHANGED_NAME_VALUE = "Test board";
+    private String CHANGED_DESC_VALUE = "Decription is empty";
+
```
Comment (SoniaMiach) [inline src/test/java/api/put/UpdateABoard.java:19]: delete empty space

**Review by SoniaMiach — APPROVED** [review]

File: src/test/resources/application.conf (hunk 1 above)
Comment (Goldbe54) [reply src/test/resources/application.conf:2]: I can't run the tests with the previous token and key

File: src/test/java/api/put/UpdateABoard.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import org.testng.annotations.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class UpdateABoard extends BaseTest {
+    private String NAME_VALUE = "Test board";
+    private String DESC_VALUE = "Decription is empty";
+    private String CHANGED_NAME_VALUE = "Test board";
+    private String CHANGED_DESC_VALUE = "Decription is empty";
```
Comment (KolyaSamandasiuk) [inline src/test/java/api/put/UpdateABoard.java]: add final keyword

**Review by KolyaSamandasiuk — APPROVED** [review]

**Pull request merged**